issue_release_status.go

```diff
@@ -80,13 +81,18 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 	req.Header.Add("Authorization", "Bearer "+handler.releaseManagerAuthToken)
 
 	resp, err := client.Do(req)
+	body, err := ioutil.ReadAll(resp.Body)
+
 	if err != nil {
 		return errors.Wrap(err, "sending HTTP request")
 	}
+	if resp.StatusCode != 200 {
+		logger.Info().Msgf("Request body: %v", body)
+		return errors.Wrap(err, "expected status code 200, but recieved "+fmt.Sprintf("%v", resp.StatusCode))
```

Comment:
As `err` is actually `nil` in this case, this will silently fail.

If you use `errors.Errorf()` instead, we create a new error to be returned.